javascript/app-event-analysis/src/App/App.hooks.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+
+import type { PivotItem } from '@fluentui/react'
+import { useTheme } from '@fluentui/react'
+import { isEmpty } from 'lodash'
+import { useCallback, useEffect, useMemo, useRef } from 'react'
+
+import { useProcessedInputData } from '../hooks/useProcessedInputData.js'
+import {
+	useColumnMappingState,
+	useSetSelectedTabKeyState,
+	useTreatedUnitsState,
+	useTreatmentStartDatesState,
+} from '../state/index.js'
+
+export function usePivotStyles() {
```

Comment:
I've been moving to a model where hooks that only generate styles are contained in the Component.styles.ts file